### PR TITLE
Added two attributes which allow to block manual member adding

### PR DESCRIPTION
**Pull request by HejdaJakub**
* I added one attribute for Vo and one for Group, which allow to disable manually add members to these entities.

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java [hunk 1]
```diff
@@ -7284,7 +7284,30 @@ protected void initialize() {
 		rights.add(new AttributeRights(-1, Role.VOADMIN, Arrays.asList(ActionType.READ, ActionType.WRITE)));
 		attributes.put(attr, rights);
 
+		//urn:perun:vo:attribute-def:def:blockManualMemberAdding
+		attr = new AttributeDefinition();
+		attr.setDisplayName("Block manual member adding");
+		attr.setFriendlyName("blockManualMemberAdding");
+		attr.setNamespace(AttributesManager.NS_VO_ATTR_DEF);
+		attr.setDescription("Allows to disable manually add members to Vo.");
```
Comment (zlamalp) [inline perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java:7292]: How about something more straightforward and explanatory, like: ``Block manual addition of Vo members. Users are expected to go through registration.`` (same for the group case).

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java [hunk 2]
```diff
@@ -7284,7 +7284,30 @@ protected void initialize() {
...
+		attr.setType(Boolean.class.getName());
+		//set attribute rights (with dummy id of attribute - not known yet)
+		rights = new ArrayList<>();
+		rights.add(new AttributeRights(-1, Role.VOADMIN, Arrays.asList(ActionType.READ, ActionType.WRITE)));
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java:7296]: Done.

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_blockManualMemberAdding.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.GroupAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.GroupAttributesModuleImplApi;
+
+/**
+ * @author Jakub Hejda <Jakub.Hejda@cesnet.cz>
+ */
+public class urn_perun_group_attribute_def_def_blockManualMemberAdding extends GroupAttributesModuleAbstract implements GroupAttributesModuleImplApi {
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_blockManualMemberAdding.java]: Modules have been deleted.

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java (hunk 1 above)
Comment (HejdaJakub) [reply perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java]: Done.

**Review by zlamalp — APPROVED** [review]

**Pull request merged**